apps/dashboard/src/components/editor/index.js

```diff
@@ -76,4 +57,10 @@ const Editor = ( { projectId } ) => {
 
 registerBlocks();
 
-export default Editor;
+export default withSelect( ( select, ownProps ) => {
```

Comment:
Side note: hooks are generally preferred to HoC's as they're slightly more readable and performant.  
However, a good use of HoC's would be when you want to export the 'bare' component as well as a version connected to the store - like with the `<Masterbar />` for whatever purpose.

Anyway, let's keep this now but use hooks when possible in the future for consistency.